### PR TITLE
Add `all` parameter to `show/clear queue watermark` command

### PR DESCRIPTION
**Pull request by bratashX**
Signed-off-by: Petro Bratash <petrox.bratash@intel.com>
Depends on Azure/sonic-swss#1653

**- What I did**
Add new commands :
`show  queue persistent-watermark all`
`show  queue watermark all`
`sonic-clear queue watermark all`
`sonic-clear queue persistent-watermark all`

**- How I did it**
Add **_q_shared_all_**  type  to  watermarkstat file
**- How to verify it**
Execute:
`show  queue persistent-watermark all`
`show  queue watermark all`
`sonic-clear queue watermark all`
`sonic-clear queue persistent-watermark all`

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

**Comment by bratashX** [comment]
@akokhan, please review

**Review by akokhan — APPROVED** [review]

File: scripts/watermarkstat [hunk 1]
```diff
@@ -254,6 +264,8 @@ def main():
 Examples:
   watermarkstat -t pg_headroom
   watermarkstat -t pg_shared
+  watermarkstat -t q_shared_all
+  watermarkstat -p -t q_shared_all
```
Comment (neethajohn) [inline scripts/watermarkstat:268]: For platforms that do not have this queue type, it will error out. Please handle that case

Comment (bratashX) [reply scripts/watermarkstat:268]: If platforms not have this queue type, command print only port name, queue table will be empty.
> $ show queue watermark all
> Egress shared pool occupancy per all queues:
> Port
> \-----------
> Ethernet0
> Ethernet4
> Ethernet8
> Ethernet12
> Ethernet16
> Ethernet20
> Ethernet24
> Ethernet28
> ......

**Comment by lguohan** [comment]
for all new feature, unit tests are required.

**Review by lguohan — DISMISSED** [review]
for all new feature, unit tests are required.

**Comment by bratashX** [comment]
Unit tests for this feature were added.
@akokhan @neethajohn @lguohan, please review

**Comment by bratashX** [comment]
@neethajohn @lguohan, please review

**Comment by bratashX** [comment]
@neethajohn @lguohan, please review and merge

**Comment by neethajohn** [comment]
@bratashX , Is there a PR in swss to handle the clear request for 'Q_SHARED_ALL'.? Otherwise the clear command will result in 
https://github.com/Azure/sonic-swss/blob/master/orchagent/watermarkorch.cpp#L221

**Comment by bratashX** [comment]
> @bratashX , Is there a PR in swss to handle the clear request for 'Q_SHARED_ALL'.? Otherwise the clear command will result in
> https://github.com/Azure/sonic-swss/blob/master/orchagent/watermarkorch.cpp#L221

@neethajohn Thanks, related PR in sonic-swss: https://github.com/Azure/sonic-swss/pull/1653

**Comment by bratashX** [comment]
@lguohan @neethajohn could you please review and merge

**Review by neethajohn — APPROVED** [review]

**Comment by bratashX** [comment]
@neethajohn @lguohan  could you please merge? Thanks

**Comment by bratashX** [comment]
@neethajohn  thanks for merge https://github.com/Azure/sonic-swss/pull/1653
Please also merge  this PR,  because it related

**Pull request merged**